bump-version/index.js

```diff
@@ -32,6 +32,29 @@ else {
     console.log(`Making a "${version}" version bump ...`)
 
     var res
+	var manifest
+	var newVersion
+	
+	if (baseDirectory != '' && baseDirectory != '.') {
+        workdir += `/${baseDirectory}`
+    }
+	
+	if (utils.fileExists(workdir + '/manifest.yaml')) {
+        manifest = 'manifest.yaml'
+    } else if (utils.fileExists(workdir + '/manifest.yml')) {
+        manifest = 'manifest.yml'
+    } else if (utils.fileExists(workdir + '/manifest.json')) {
+        throw new Error('Bump version on manifest.json is not supported yet.')
+    } else {
+        throw new Error('No manifest found.')
```

Comment:
This script may be used for projects which don't have a manifest - can we warn on manifest not found instead?